wordpress-mobile/WordPress-iOS
# Fix/16941 - Close account displayed localizable string

This PR changes localizable string displayed when close account error occurs due to having atomic site to a more generic one, with the instruction to contact support.
This change is needed because having an atomic site is something that's considered being an implementation detail user might not necessarily know about.

## To test:
1. Make sure the current user has an atomic site
2. Under **My Site** tab tap on profile button at the top right corner
3. Tap on **Account Settings**
4. Tap on **Close account**
5. Close account alert is displayed

## Screenshot:
<img width="509" alt="Screen Shot 2021-08-24 at 01 56 30" src="https://user-images.githubusercontent.com/12729242/130535269-4f5b2f05-01c7-43be-8393-9c3c02d10065.png">

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a
PR submission checklist:
